cmd/bootstrapper/generator.go

```diff
@@ -56,23 +57,42 @@ func WithFilesystem(fs afero.Fs) Opt {
 	}
 }
 
-func WithHttpClient(c *http.Client) Opt {
+func WithHttpClient(c *retryablehttp.Client) Opt {
 	return func(g *Generator) {
 		g.client = c
 	}
 }
 
+func checkRetry(ctx context.Context, resp *http.Response, err error) (bool, error) {
+	if resp != nil && resp.StatusCode != http.StatusOK {
+		return true, nil
+	}
+	return retryablehttp.DefaultRetryPolicy(ctx, resp, err)
+}
```

Comment:
I'm not sure, should it retry on every possible failure? I think we could stick with `retryablehttp` defaults, they are sane, and change this only if needed.

Reply:
switched to use default policy